doc/source/whatsnew/v2.2.1.rst

```diff
@@ -19,6 +19,7 @@ Enhancements
 
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
+- :meth:`DataFrame.__dataframe__` was producing incorrect data buffers when the a column's type was a pandas nullable on with missing values (:issue:`56702`)
```

Comment:
Oh actually. I guess we need a 2.2.2 whatsnew first

Reply:
ah right, 2.2.1 is already out, sorry about that, thanks for spotting it